pkg/apis/wksprovider/controller/wksctl/machine_controller.go

```diff
@@ -300,15 +302,16 @@ func (a *MachineController) installNewBootstrapToken(ctx context.Context, ns str
 	if err != nil {
 		return nil, gerrors.Errorf("failed to create new bootstrap token %s/%s", ns, secret.ObjectMeta.Name)
 	}
-	s, err := a.clientSet.CoreV1().Secrets(ns).Create(ctx, secret, metav1.CreateOptions{})
+	err = a.client.Create(ctx, secret)
 	if err != nil {
 		return nil, gerrors.Errorf("failed to install new bootstrap token %s/%s", ns, secret.ObjectMeta.Name)
 	}
+	s := secret
```

Comment:
This assignment seems a bit redundant since `secret` is a pointer type.

Reply:
Yeah it's to reduce diffs to make it easier to see what really changed. 
I can add another commit to squash the redundant stuff.

Reply:
Actually I remember now; this code will change again when we fix #288 so I thought I'd just leave it for now.